src/wp-includes/block-supports/duotone.php

```diff
@@ -57,3 +57,7 @@
 
 // Migrate the old experimental duotone support flag.
 add_filter( 'block_type_metadata_settings', array( 'WP_Duotone', 'migrate_experimental_duotone_support_flag' ), 10, 2 );
+
+// Restore duotone classes to the image wrapper div.
+add_filter( 'render_block_core/image', array( 'WP_Duotone', 'restore_image_outer_container' ), 11, 2 );
```

Comment:
```suggestion
add_filter( 'render_block_core/image', array( 'WP_Duotone', 'restore_image_outer_container' ), 10, 1 );
```

Reply:
I think we might have to keep the higher priority because the filter isn't working on your other PR @ajlende, but it's working on this one.

Reply:
I forgot that this was weird in the PR. Things have to be loaded in a very specific order. https://github.com/WordPress/gutenberg/pull/59764#issuecomment-1992211426

> When this gets backported to core, the same should happen with the core version of the functions. Although, it looks like the imports will have to be reordered during the core backport ([WordPress/wordpress-develop@9a616a5/src/wp-settings.php#L371](https://github.com/WordPress/wordpress-develop/blob/9a616a573434b432a5efd4de21039250658371fe/src/wp-settings.php#L371)).
>
> In my opinion, with the way WordPress code is written with side effects in many files, the way that we mostly centralize imports, the size of the codebase, and the number of contributors, it makes more sense for use to just preserve the order of imports when backporting things to not have to worry so much about priority levels. It seems better to me to maintain that order than force extenders to use extra high priorities to override core behavior.

Reply:
Cool we can go with that one then!